alerta-covid/spec/requests/api/v1/areas_spec.rb

```diff
@@ -10,19 +10,20 @@
   }
 
   before(:each) do
+    FactoryBot.create(:institution, id:1, name: "institution")
```

Comment:
You shouldn't specify the `id`

Any particular reason for doing it?

Reply:
ready, now commits based on these observations